python/lsst/ts/mtrotator/rotator_csc.py

```diff
@@ -552,6 +592,28 @@ def make_mock_controller(self, initial_ctrl_state):
             initial_state=ControllerState.STANDBY,
         )
 
+    @classmethod
+    def add_arguments(cls, parser: argparse.ArgumentParser) -> None:
+        super(RotatorCsc, cls).add_arguments(parser)
+
+        parser.add_argument(
+            "--bypass-ccw",
+            action="store_true",
+            default=False,
+            help="""
+                 Bypass the check of camera cable wrapper (CCW) or not. This is
+                 for the test purpose only.
```

Comment:
Consider rephrasing it:

> This is for testing purpose only.

Reply:
Fixed in the commit: [f03b120](https://github.com/lsst-ts/ts_mtrotator/pull/82/commits/f03b120f7da830090949ea0b1a365e87fd3689b1). Thanks!